cmd/main.go

```diff
@@ -264,6 +268,16 @@ func main() {
 	bm := bminventory.NewBareMetalInventory(db, log.WithField("pkg", "Inventory"), hostApi, clusterApi, Options.BMConfig,
 		generator, eventsHandler, objectHandler, metricsManager, *authHandler, ocpClient)
 
+	if isLeader {
```

Comment:
There is a way to do it with leader election, please take a look how other monitors are doing it